test/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_subdirectory(backend)
 # The lit and unit tests do NOT require ONNX package installation
 add_subdirectory(mlir)
 add_subdirectory(numerical)
+add_subdirectory(extra)
```

Comment:
I really don't like the name `extra` since it's not descriptive. Is there something more descriptive that we could use instead?

Reply:
I don't really like it much either :-) Open to suggestions.  I can think of:
  - regression (fo regression tests - in the sense of preventing regressions)
  - unittest

Reply:
`unittest` is probably best, though `backend-cpp` is not bad either.

Reply:
fine with either

Reply:
will use `backend-cpp`